src/Service/ZGWDocumentToFileService.php

```diff
@@ -0,0 +1,385 @@
+<?php
+/**
+ * An example service for adding business logic to your class.
+ *
+ * @author  Conduction.nl <info@conduction.nl>
+ * @license EUPL-1.2 https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
```

Comment:
Please move this block down to just above:
/** here */
class ZGWDocumentToFileService